app/src/main/java/org/fossasia/openevent/general/utils/DataBindingAdapters.kt

```diff
@@ -0,0 +1,65 @@
+package org.fossasia.openevent.general.utils
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import com.squareup.picasso.Callback
+import com.squareup.picasso.Picasso
+import org.fossasia.openevent.general.CircleTransform
+import org.fossasia.openevent.general.R
+import timber.log.Timber
+
+object DataBindingAdapter {
```

Comment:
No need of this, just write bare functions in this file

Reply:
Bare Functions won't work as they need to be annotated with JvmStatic and for that, we need singleton class

Reply:
No they don't